DEVELOPMENT.md

```diff
@@ -0,0 +1,39 @@
+# Development Tips
+
+## Code Generation
+For matching interface between SDK and Henesis API server, SDK uses `codegen`. So, if you add/update Henesis API server, run command to generate interface codes automatically. 
+```bash
+# in pakcages/core
+npm run codegen
+```
+The interface codes are generated in `packages/core/src/__generate__`.
+
+## HTTP Headers
+When you run the API Proxy and want to call API for Henesis Wallet, do not forget to write secert to header `X-Henesis-Secret` and access token to authorization header using `Bearer`. Secert and access token could be checked in Henesis Wallet dashboard. 
+
+## Test Environment
+You can test the API Proxy with various test environments. Also, you can choose the server environment editing some envionment variables.
```

Comment:
How about adding "by" before the "editing"?
Adding "by" would make the relationship of two pharses clear.
"you can choose the server environment editing some envionment variables" -> "you can choose the server environment by editing some envionment variables"